### PR TITLE
add application-level statistics

### PR DESCRIPTION
**Pull request by bigerous**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: scheduler/metrics/metrics.go [hunk 1]
```diff
@@ -56,28 +56,28 @@ var (
...
-	}, []string{"tag"})
+	}, []string{"tag", "biz_app"})
 
 	DownloadFailureCount = promauto.NewCounterVec(prometheus.CounterOpts{
 		Namespace: constants.MetricsNamespace,
 		Subsystem: constants.SchedulerMetricsName,
 		Name:      "download_failure_total",
 		Help:      "Counter of the number of failed of the downloading.",
-	}, []string{"tag", "type"})
+	}, []string{"tag", "biz_app", "type"})
```
Comment (gaius-qi) [inline scheduler/metrics/metrics.go:80]: Replace `biz_app` to `application`.

**Review by gaius-qi — CHANGES_REQUESTED** [review]

File: scheduler/metrics/metrics.go (hunk 1 above)
Comment (bigerous) [reply scheduler/metrics/metrics.go]: replaced with app

File: scheduler/resource/peer.go [hunk 2]
```diff
@@ -110,13 +113,24 @@ func WithTag(tag string) PeerOption {
...
+}
+
 type Peer struct {
 	// ID is peer id.
 	ID string
 
 	// Tag is peer tag.
 	Tag string
 
+	// Application is peer application
```
Comment (gaius-qi) [inline scheduler/resource/peer.go:131]: This sentence is missing a `.` ending.

File: scheduler/resource/seed_peer.go [hunk 3]
```diff
@@ -32,7 +32,8 @@ import (
 
 const (
 	// Default value of biz tag for seed peer.
-	SeedTag = "d7y/seed"
+	SeedTag         = "d7y/seed"
+	SeedApplication = "d7y-seed"
```
Comment (gaius-qi) [inline scheduler/resource/seed_peer.go:36]: Const requires remarks. Consistent with the tag => `d7y/seed`, no need to redefine.

**Review by gaius-qi — CHANGES_REQUESTED** [review]

**Review by gaius-qi — APPROVED** [review]
LGTM

**Review by jim3ma — APPROVED** [review]

**Pull request merged**